src/hooks/useAlertsPageData.js

```diff
@@ -42,8 +42,8 @@ export const useAlertsPageData = (filters, isAlertsPage) => {
 
       const projectName = !isAlertsPage
         ? params.projectName || params.id
-        : filters?.[PROJECTS_FILTER]?.toLowerCase?.() !== FILTER_ALL_ITEMS
-          ? filters?.[PROJECTS_FILTER]?.toLowerCase?.()
+        : filters?.[PROJECT_FILTER]?.toLowerCase?.() !== FILTER_ALL_ITEMS
+          ? filters?.[PROJECT_FILTER]?.toLowerCase?.()
```

Comment:
Filter by project still won't work correctly if the project's name will be `all`
We allow project with this name. So, this case must be handled as well